### PR TITLE
#1237 add privacy policy page for crowdsource app

### PR DESCRIPTION
**Pull request by nya-elimuai**
Added privacy policy page since this is required for publishing the [Crowdsource](https://github.com/elimu-ai/crowdsource) app on Google Play.

Screen Shot 2021-04-08 at 2 37 11 PM

### CODE REVIEW

File: src/main/webapp/WEB-INF/jsp/contributions/privacy-policy.jsp [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        the privacy of its community members (contributors):
+    </p>
+    
+    <ol class="browser-default">
+        <li>
+            When a community contributor signs onto this website or the 
+            <a href="https://github.com/elimu-ai/crowdsource" target="_blank">Crowdsource</a> 
+            Android app for the first time, we store the anonymous provider ID (from 
+            Google/GitHub) in order to recognize the same contributor when 
+            she returns at a later point in time.
```
Comment (Keerthi4308) [inline src/main/webapp/WEB-INF/jsp/contributions/privacy-policy.jsp:19]: nitpick: Should we consider: he/she here?

File: src/main/webapp/WEB-INF/jsp/contributions/privacy-policy.jsp [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        </li>
+        <li>
+            We keep track of each contributor's contributions. As an example, each 
+            time a contributor uploads a word's audio recording to the website, 
+            we store information about this event. The reason for doing this is 
+            so that other community members can peer review the content and 
+            quality approve the contribution.
+        </li>
+        <li>
+            A community member can request a deletion of her account and any
```
Comment (Keerthi4308) [inline src/main/webapp/WEB-INF/jsp/contributions/privacy-policy.jsp:33]: nitpick: Should we consider: his/her here?

File: src/main/webapp/WEB-INF/jsp/contributions/privacy-policy.jsp (hunk 1 above)
Comment (nya-elimuai) [reply src/main/webapp/WEB-INF/jsp/contributions/privacy-policy.jsp]: > nitpick: Should we consider: he/she here?

@Keerthi4308 Fixed 🙂

**Review by Keerthi4308 — APPROVED** [review]

**Comment by So-Somila** [comment]
Accept

**Pull request merged**